package.json

```diff
@@ -21,7 +21,8 @@
     "magnet-uri": "^6.2.0",
     "queue-microtask": "^1.2.3",
     "simple-get": "^4.0.1",
-    "simple-sha1": "^3.1.0"
+    "simple-sha1": "^3.1.0",
+    "simple-sha256": "^1.1.0"
```

Comment:
congratulations, you opened pandora's box, this needs to be deprecated in favor of uint8-utils async hash() function, which means making the entire program async! have fun :^)